### PR TITLE
Initial changes for EG Blob trigger

### PR DESCRIPTION
**Pull request by castrodd**
What does this implement/fix? Explain your changes.
---------------------------------------------------
…

Does this close any currently open issues?
------------------------------------------

Any relevant logs, screenshots, error output, etc.?
-------------------------------------

Any other comments?
-------------------
…

Has this been tested?
---------------------------
- [X] Tested

### CODE REVIEW

File: azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/resources/bindings.json [hunk 1]
```diff
@@ -91,6 +91,37 @@
         }
       ]
     },
+    {
+      "type": "blobTrigger",
```
Comment (Flanker32) [inline azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/resources/bindings.json]: please change `type` to `EventGridBlobTrigger` to join with the `templates.json`

Comment (castrodd) [reply azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/resources/bindings.json]: Updated `type` name

**Review by Flanker32 — DISMISSED** [review]
LGTM

**Review by Flanker32 — APPROVED** [review]

**Review by wangmingliang-ms — APPROVED** [review]
LGTM

**Pull request merged**